### DISCUSSION TITLE
How to open a file/directory after the find command finishes?

### DISCUSSION THREAD

**Discussion by joaopauloalbq** [gokcehan/lf#1456] [Q&A] [answered]
I'm using:
`set incsearch true` # to jump to the current match at each keystroke while typing
`set findlen 0` # For the command to end when there is only a single match

I tried `cmap f :find; open` but it didn't work :/

**Comment by DusanLesan** [comment, marked as answer]
Looks related to this: https://github.com/gokcehan/lf/issues/1082
So maybe something like this would work for you:
```
map f :find; set user_find true
cmd on-select &{{
	if [ "$lf_user_find" = true ]; then
		lf -remote "send $id set user_find false"
		lf -remote "send $id open"
	fi
}}
```

**Reply by joaopauloalbq** [reply]
That's it, thank you very much!
